youtube_dl/extractor/googledrive.py

```diff
@@ -44,6 +51,10 @@ class GoogleDriveIE(InfoExtractor):
         '46': 'webm',
         '59': 'mp4',
     }
+    _SUBTITLE_FORMATS_EXT = (
+        'vtt',
+        'ttml'
+    )
```

Comment:
Whoops, never mind - I didn't read your comment properly :)

An updated PR will be submitted shortly.  Thanks for the follow up!